presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/CachingJdbcClient.java

```diff
@@ -55,7 +55,7 @@
     private final JdbcClient delegate;
     private final boolean cacheMissing;
 
-    private final LoadingCache<JdbcIdentity, Set<String>> schemaNamesCache;
+    private final LoadingCache<ConnectorSession, Set<String>> schemaNamesCache;
```

Comment:
ConnectorSession is not a good candidate for the cache key and we avoided it so far